tkcalendar.py

```diff
@@ -118,6 +127,9 @@ def __init__(self, master=None, **kw):
                 self._sel_date = None
 
         self._date = self.date(year, month, 1)  # (year, month) displayed by the calendar
+        
+        if self._textvariable is not None:
+          self._textvariable.set(self._sel_date.strftime("%x"))
```

Comment:
This will throw an error if `self._sel_date` is `None`, you should move this line just below line 125.